e2e/scenario/scenario_ubuntu2204-custom-ca-trust.go

```diff
@@ -0,0 +1,37 @@
+
+func ubuntu2204CustomCATrust() *Scenario {
+	return &Scenario{
+		Name:        "ubuntu2204-custom-ca-trust",
+		Description: "Tests that a node using the Ubuntu 2204 VHD can be properly bootstrapped and custom CA was correctly added",
+		Config: Config{
+			ClusterSelector: NetworkPluginKubenetSelector,
+			ClusterMutator:  NetworkPluginKubenetMutator,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.CustomCATrustConfig = &datamodel.CustomCATrustConfig{
+					CustomCATrustCerts: []string{
+						encodedTestCert,
+					},
+				}
+			},
+			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {
+				vmss.Properties.VirtualMachineProfile.StorageProfile.ImageReference = &armcompute.ImageReference{
+					ID: to.Ptr(DefaultImageVersionIDs["ubuntu2204"]),
+				}
+			},
+			LiveVMValidators: []*LiveVMValidator{
+				FileExistsInDirectoryValidator("/usr/local/share/ca-certificates/certs"),
```

Comment:
why can't we use DirectoryValidator instead?

Reply:
From what I understand DirectoryValidator checks for specific file name, the way CustomCA works is it generates a new name for file after copying, I could technically hardcode that name there, but all I really care about in this test is that one file exists in this dir.